src/parser/jobs/rdm/modules/Dualcast.js

```diff
@@ -36,9 +36,8 @@ export default class DualCast extends Module {
 	//The last timestamp for a change in CastType
 	_castTypeLastChanged = null
 	_severityMissedDualcast = {
-		1: SEVERITY.MINOR,
-		2: SEVERITY.MEDIUM,
-		3: SEVERITY.MAJOR,
+		1: SEVERITY.MEDIUM,
+		2: SEVERITY.MAJOR,
```

Comment:
Any reason major is also changing here?

Reply:
The reasoning was that short of an invuln trigger preventing you from casting there's no reason to waste a dualcast, whether you use it on the proper spell or anything else it should never just fall off.  Thus if you let two drop that's a major problem you should look at.

Reply:
It should be noted that the miss logic does handle for invuln.